RSSDP/SsdpDevicePublisher.cs

```diff
@@ -427,7 +438,12 @@ private void SendAliveNotification(SsdpDevice device, string notificationType, s
 
             var message = BuildMessage(header, values);
 
-            _CommsServer.SendMulticastMessage(message, cancellationToken);
+            if (_sendOnlyMatchedHost)
+            {
+                _CommsServer.SendMulticastMessage(message, _sendOnlyMatchedHost ? rootDevice.Address : null, cancellationToken);
```

Comment:
huh, I proposed to replace the whole `if..else` with this ternary, it's just I cannot leave a suggestion for more than one line in GH :)

Reply:
Sorry, have some carelessness yesterday, will append one more comment to update the code soon,